ganymede/models/stahl/cd_locate_stores.sql

```diff
@@ -0,0 +1,9 @@
+{{ config(materialized='view') }}
+
+with store_locations as (
+    select id, r_store_attributes->'geo'->'latitude' as latitude, r_store_attributes->'geo'->'longitude' as longitude, {{ raw_region_to_province("store_address->>'addressRegion'") }} as region
+    from {{ source('aethervest', 'stores') }}
+)
+select store_locations.id, cds.cdname, store_locations.region 
+from store_locations inner join {{ source('static_resources', 'censusdivisions') }} cds
```

Comment:
```suggestion
from store_locations inner join {{ source('static', 'census_divisions') }} cds
```

If you change this to a left join, you can then make an assertion that cdname is not null in the test

Reply:
Just so I understand (kinda new to the fancier kinds of sql joins)--using a left join we can ensure that if for some reason a store cannot be located within a census division then that store record will have null in its cdname column?

Reply:
Yes, and this way, we can write something like this in the dbt test - 

```
models:
  - name: dim_store
    columns:
      - name: census_division_name
        tests:
          - not_null
```

Speaking of which, you should add the model to the `.yml` ... I generally tend to not use acronyms if possible btw e.g. (cd_name)  ... especially if the resource is to be bound to the api, which this prolly will. This way, there's minimum relabeling in the API and the end-user blind to this part of the repo has some understanding of the definitions without having to read to many docs.

Reply:
Got it

Seems like there currently is no `stahl.yml` file, should I make one?